gallery/src/pageConfig.ts

```diff
@@ -123,6 +124,7 @@ const pageConfig: PageConfig = {
   'Styles - Mixins & Helpers': {
     'nx-clickable': NxClickablePage,
     'nx-container-helpers': NxContainerHelpersPage,
+    'nx-font-size': NxFontSizePage,
```

Comment:
Should this also be adjusted to match its current name?

Reply:
> Should this also be adjusted to match its current name?

"font-size" did not seem like a very helpful name so I changed it to "Custom app font size": https://github.com/sonatype/sonatype-react-shared-components/pull/92/commits/1feeffdc9a085094d55cbb5682c76fdd494a3c3a